Milestone #1

Bismarck-GM opened the pull request:
## Milestone 1
- Studied the layout of Smashing-Magazine website.
- Created a basic layout with HTML and CSS.
- Added color gradients to page elements.
- Formated the stylings so it would look similar to the original.

Bismarck-GM committed:
Added Index CSS and Readme

Started building the HTML layout

Bismarck-GM committed:
Started layout with Grid and Flex

Bismarck-GM committed:
Finished placing HTML elements and color layout

simandebvu committed:
Added heatmap colors for whole page

simandebvu committed:
Added Fonts, logo and navbar flexbox

Bismarck-GM committed:
Added Stylelintconfig.js and .stickler.yml

Edited readme file.

Bismarck-GM committed:
Finished navbar

Bismarck-GM committed:
Added and styled mockup content

All with visual height coloring.

simandebvu committed:
Styled the community articles

simandebvu committed:
Curated posts styled

Bismarck-GM committed:
Styled newsletter section

Bismarck-GM committed:
Finished Footer and Newsletter section

Bismarck-GM committed:
Finished article publicity items

simandebvu committed:
Styled and Resolved HTML Validation Errors

simandebvu committed:
Resolved CSS Stylelint Errors

Bismarck-GM committed:
Added README link to live and project details

Bismarck-GM committed:
Corrected link to screenshot

Bismarck-GM committed:
Corrected screenshot link again

Bismarck-GM committed:
Fixed fonts.css lint errors

addod19 reviewed (changes requested):
## Status: Require changes :recycle: :traffic_light: 

Hi @Bismarck-GM and @simandebvu, you guys have done a great job :+1: The project looks great, the README needs a little instruction on cloning the project and the PR has a title and a description. However, to improve our project and make it even better, let us address the comments below:
- Files not related to the project should be added to the .gitignore file
- Kindly fix the horizontal scroll
- ```Nav``` uses div to position elements
![na](https://user-images.githubusercontent.com/46388113/81871507-7a0c0680-9567-11ea-8874-086f97ab64f4.png)
- ```2 Article rows```, In this case, we are expecting 2 rows with each row having 2 articles. Each article takes 1/2 width of the screen
![sh1](https://user-images.githubusercontent.com/46388113/81871714-e8e95f80-9567-11ea-9daa-3956c7e6bb4b.png)
- ``` 3 Article rows```, In this case, we are expecting 3 rows, each row having 3 articles and each article having a width of 1/3 of the screen
![sh2](https://user-images.githubusercontent.com/46388113/81871955-56958b80-9568-11ea-8570-7b4d10772c00.png)
- Kindly fix the warning returned by the w3validator
![sh3](https://user-images.githubusercontent.com/46388113/81872137-b8ee8c00-9568-11ea-8156-f82feab28d4b.png)
- Kindly apply the DRY rule to our CSS file. This [article](https://speckyboy.com/good-bad-css-practices/) could help
- Kindly implement the CSS float property
![tw](https://user-images.githubusercontent.com/46388113/81872410-34e8d400-9569-11ea-8577-363350ffb93e.png)

Kindy implement these changes and submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo)

Bismarck-GM committed:
Added gitignore and media queries to manage resolution below F-HD

Added rules to CSS

Bismarck-GM committed:
Added div to main-nav and fixed validator warning

Bismarck-GM committed:
Applied DRY rule. Implemented Float. Fixed horizontal scroll overflow.

Bismarck-GM committed:
Edited readme

Bismarck-GM commented:
## Milestone 2:
- Added media queries to manage the number of columns within the first 2 sections.
- Fixed horizontal overflow.
- Added helper classes to CSS and HTML. 
- Fixed validator warning.

trekab reviewed (approved):
# Approved! :tada: 
Well done @Bismarck-GM and @simandebvu! :clap: You have done an impressive job! You may now merge to master, submit the project completion form and continue to your next project.

Happy coding! :smile: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)
